packages/cli/src/lib/cmds/init.ts

```diff
@@ -81,6 +81,14 @@ async function confirmTwaConfig(twaManifest: TwaManifest): Promise<TwaManifest>
       default: twaManifest.maskableIconUrl,
       filter: (input): string | undefined => input.length === 0 ? undefined : input,
       validate: async (input): Promise<boolean> => input === undefined || await validateUrl(input),
+    }, {
+      name: 'monochromeIconUrl',
+      type: 'input',
+      message: 'URL to an image that is at least 48x48px to be used when generating ' +
+          'monochrome icons',
```

Comment:
Do you reckon we need a little more explanation here as to what the monochrome icon is? Specifically that it doesn't actually need to be monochrome, it just needs to look good when turned monochrome.

Also, who determines which colour it will be displayed.

(I know these are in the spec, but should we maybe link to that or just include a sentence of explanation?)

Reply:
My understanding is that we are using monochrome icons in the Web Manifest to generate notification icons in the Android App. It sounds to me that we inside the Bubblewrap we should call this "notification icon"?